### ISSUE TITLE
Setting of hyper parameter

### ISSUE THREAD

**Issue by wzhSteve** [ICTMCG/FakingRecipe#3]
Hi, I can't find the setting of the learning rate, alpha and beta in the loss function in either the paper or the code. Could you please provide details on these three hyperparameters? Thanks a lot.

**Comment by ghh1125** [comment]
I also want to know these setting of the learning rate, alpha and beta

**Comment by Yuyan-B** [comment]
> Hi, I can't find the setting of the learning rate, alpha and beta in the loss function in either the paper or the code. Could you please provide details on these three hyperparameters? Thanks a lot.

We set the learning rates of 5e-5 for FakeSV and 1e-3 for FakeTT. For hyperparameter alpha and beta,  we search from 0.1 to 3 and set them at 0.1 and 2.0, respectively.
